### DISCUSSION TITLE
modflow_coupling error

### DISCUSSION THREAD

**Discussion by whq552800** [iiasa/CWatM#125] [Q&A] [answered]
when i coupled modflow with cwatm, the result only shows the MODFLOW outputs.And i can't find the error.The settings file and the cmd screen uploaded below.The template i used is "CWatM-Earth-30min",and the part of modflow came from "[cwatmmodflow6_github.zip](https://github.com/iiasa/CWatM/files/7569445/cwatmmodflow6_github.zip)" 
Thanks!
Wang haiqing.
![fba80f596a2102b6f8219a8188ed3a6](https://github.com/iiasa/CWatM/assets/120584339/907ddf21-1611-42de-b91a-129835145795)
[settings_file.txt](https://github.com/iiasa/CWatM/files/15137971/settings_file.txt)

**Comment by dof1985** [comment]
Hi Wang,

Thanks for reaching out. Modflow can be a bit tricky sometimes. Please provide the 'TRANSIENT.lst' file from this folder: PathGroundwaterModflowOutput = F:\cwatm_output\north_east4\dbmodflow

**Reply by whq552800** [reply]
I'm sorry, but the output floder doesn't have the 'TRANSIENT.lst' file,it just output 16 files.And there is two probem i have met. In 'preprocessing_forModFlow' section,i have changed "gdf['DN']" into gdf['geometry'] in "create_raster.py", and remove "libmf6.dll" and "mf6.exe" into "D:\Documents\GitHub\CWatM\cwatm"floder by the error. Could these changes make wrong?
![image](https://github.com/iiasa/CWatM/assets/120584339/7ab9b18e-04d1-4288-b888-508cad8fb843)
![image](https://github.com/iiasa/CWatM/assets/120584339/9591b982-06b8-4caa-95c5-d02bb14b9680)
![image](https://github.com/iiasa/CWatM/assets/120584339/42d3a4aa-11c5-4e40-ad98-957a77df4ab3)

**Comment by dof1985** [comment, marked as answer]
Please check that your 'xmipy' version is set to 1.0.0. The modflow coupling necessitates this version.

**Reply by whq552800** [reply]
Yes, that's the problem! Thanks again for you apply.

**Comment by weixiaoxun** [comment]
haiqing，您好，我能请教您一些问题吗？
                                               魏

**Reply by whq552800** [reply]
可以，你说

**Reply by weixiaoxun** [reply]
> 可以，你说

您好，王师兄。我是河南大学地理与环境学院2023级的一名研究生。
我最近在研究modflow耦合，我试着用教授的例子bhima，在坤师兄的帮助下我跑通了，但我替换成自己区域的时候出了问题（ldd和maskmap）。所以我想请教您这一解决办法，谢谢，中秋节快乐。
我的微信是weilinze

**Reply by fwh1735148251** [reply]
> 可以，你说
您好 ，王老师，我最近有一个 CWatM + MODFLOW 6 地表地下水耦合方面的任务，查资料时看到您之前实际做过 CWatM-MODFLOW，所以想向您有偿求助一下。

任务主要是华北平原区域的耦合模拟、水资源配置和率定验证，不是免费请教，可以按任务付费。

不知道您近期有没有时间？如果方便，我把参考论文和详细要求发您先看一下，您觉得能做的话我们再谈费用和时间。

微信：fff17351

**Reply by whq552800** [reply]
你好，
不好意思，最近我比较忙，可能没有时间
抱歉
